backend/app/modules/modules_factory/protocols.py

```diff
@@ -241,3 +242,11 @@ def recalculate(self, context: Context, payload: ScoreDelegationPayload):
 
     def check(self, context: Context, addresses: list[str]) -> set[Tuple[str, str]]:
         ...
+
+
+@runtime_checkable
+class UniquenessQuotients(Protocol):
```

Comment:
do we need this Protocol if it already exists in other file?

Reply:
It's not necessary, I've just prefered adding protocols needed by a service in this service module, and protocols needed in modules factory separately in order to avoid possible circular dependencies. I'd keep it like this, but if it is confusing and there is too much boiler plate, it might be reasonable to merge all the protocols in one file. Ofc not in this PR ;)

Reply:
I can create a ticket for that if you want

Reply:
it may stay, I thought we'll get rid of uniqueness_qotient from the Pending/Current Service and it won't be needed here; after our assumptions, it's needed here indeed - i'm fine with the "boilerplate"